docs/general/explorer-integration.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: "Skip Explorer Integration"
+description: "Guide to integrating Skip Explorer v2 for transaction visualization and tracking in your applications."
+---
+
+## Overview
+
+Skip Explorer v2 provides a user-friendly interface for visualizing cross-chain transactions and tracking their progress. This guide covers basic explorer usage and how integrators can leverage it to enhance their user experience.
+
+## Basic Explorer Usage
+
+Users can view transaction details by navigating to [explorer.skip.build](https://explorer.skip.build) and entering:
+
+- **Transaction Hash**: Any transaction hash from a supported chain
+- **Chain ID**: The chain where the transaction occurred
+
+The explorer will automatically detect and display:
+- Transaction status and progress
+- Cross-chain hops and bridges used
+- Asset transfers and swaps
+- Real-time updates as transactions progress
+
+## Integration for Custom Frontends
+
+If you're building a custom frontend and want to provide users with rich transaction visualization, you can generate explorer links that include comprehensive route data.
+
+### Basic Explorer Links
+
+For simple transaction tracking, redirect users to the explorer with URL parameters:
+
+- `tx_hash`: Comma-separated list of transaction hashes
+- `chain_id`: The initial source chain ID  
+- `is_testnet`: Optional boolean parameter for testnet transactions
+
+**Example:**
+```
+https://explorer.skip.build/?tx_hash=ABC123,DEF456&chain_id=osmosis-1&is_testnet=true
+```
+
+This approach works well when you have transaction hashes but limited route context.
+
+### Advanced Rich Data Integration
+
+For a superior user experience with complete transaction context, encode your route data as base64 and pass it via the `data` parameter. This enables the explorer to display detailed multi-hop transaction flows, user addresses, and complete route information.
+
+**Example:**
+```
+https://explorer.skip.build/?data=eyJyb3V0ZSI6ey4uLn0sInVzZXJBZGRyZXNzZXMiOnsuLi59fQ==
```

Comment:
not sure if this is supposed to be a real example or not, but this definitely isn't a real one